gremlin-client/src/connection.rs

```diff
@@ -170,6 +178,64 @@ pub struct TlsOptions {
     pub accept_invalid_certs: bool,
 }
 
+#[derive(Clone, Debug)]
+pub struct WebSocketOptions {
+    /// The maximum size of a message. `None` means no size limit. The default value is 64 MiB.
+    pub(crate) max_message_size: Option<usize>,
+    /// The maximum size of a single message frame. `None` means no size limit. The limit is for
+    /// frame payload NOT including the frame header. The default value is 16 MiB.
```

Comment:
Just forwarded the defaults that were implicit from `tungstenite` to be consistent to today's behavior.